### ISSUE TITLE
Coffeescript support

### ISSUE THREAD

**Issue by laurentpayot** [codeceptjs/CodeceptJS#474]
Mocha allows me to use CoffeeScript (v2) files for my unit tests with:
```
mocha tests/unit/*_tests.coffee --compilers coffee:coffeescript/register
```
Would it be possible to have a similar option in codecept ?

PS: Thank you for your great framework that deserves more traction.

**Comment by DavertMik** [comment]
There is the same `compilers` option for `codeceptjs run`. Did you try it?

**Comment by laurentpayot** [comment]
I tried but it doesn't even recognize CoffeeScript function arrows:
```
$ codeceptjs run --compilers coffee:coffeescript/register
 
Unexpected token >
 
/home/laurent/myproject/tests/e2e/layout_tests.coffee:3
Before((I) ->
            ^
SyntaxError: 
    at createScript (vm.js:53:10)
    at Object.runInThisContext (vm.js:95:10)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at /home/laurent/.config/yarn/global/node_modules/mocha/lib/mocha.js:222:27
```
My CS code is ok because I can run my CS tests if I compile them in a directory (.js) like so:
```
coffee -b -o ./tests/e2e/.js -c ./tests/e2e/*.coffee && codeceptjs run ./tests/e2e/.js --config ../codecept.json
```
(For page objects I also had to declare `I` before `module.exports`, something you hardly do in CS, but that's fine.)

That compilation directory workaround is quite dirty…

**Comment by DavertMik** [comment]
Yes, I understand. Can you look around into source code to find how this can be solved? 
Anyway, CodeceptJS uses Mocha so the implementation for compilers should be pretty easy to add. 
I don't have much time to work on this feature but as you I'd really like to see CoffeeScript support.

**Comment by laurentpayot** [comment]
Sure, I will try to look around asap.

**Comment by laurentpayot** [comment]
I had a look but it's not obvious… Something to do with commander.js ?

**Comment by laurentpayot** [comment]
Actually it seems that all mocha command line options other than `--reporter` are not taken into account. For instance `-C` (forcing no colors) has no effect.

**Comment by laurentpayot** [comment]
I closed #538, I was using 0.5.1 and (some) mocha options seem to work in 0.6.0.
Maybe the problem is with mocha options that take an argument? (such as `--compilers`)

**Comment by laurentpayot** [comment]
Issue still here with CodeceptJS v1.0.2.
